config.json

```diff
@@ -10,6 +10,12 @@
           "label": "Config.shopkeyLabel",
           "options": {}
         },
+        "useVariants": {
+          "type": "checkBox",
+          "required": false,
+          "label": "Config.useVariantsLabel",
+          "options": {}
+        },
```

Comment:
We don't even need a new configuration here. Variants are only returned, if it's activated on our side (Admin UI).
The customer could be confused, when he activates it and nothing happens, because it's not activated within our backend.